scipy/stats/_hypotests.py

```diff
@@ -144,6 +144,222 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     return Epps_Singleton_2sampResult(w, p)
 
 
+PoissonMeansTestResult = make_dataclass('PoissonMeansTestResult',
+                                        ('statistic', 'pvalue'))
```

Comment:
For a follow up, maybe use the common result class we started to have.

Reply:
I'll take care of that here. Otherwise, you happy with this @tupui? I'd be happy for you to check that your comments were addressed before merge.